odfuzz/config.py

```diff
@@ -28,6 +28,7 @@ def __init__(self):
         self._urls_per_property = int(env_url_per_property)
         env_ignore_restriction = os.getenv(ENV_IGNORE_METADATA_RESTRICTIONS,DEFAULT_IGNORE_METADATA_RESTRICTIONS)
         self._ignore_restriction = env_ignore_restriction
+        self._method = os.getenv("ODFUZZ_HTTP_METHOD", "GET")
```

Comment:
based on this new option, the readme.md with part of available env. variables should be updated as well.

Reply:
ok